src/python/essentia/pytools/batch_extractors.py

```diff
@@ -0,0 +1,219 @@
+# Copyright (C) 2006-2020  Music Technology Group - Universitat Pompeu Fabra
+#
+# This file is part of Essentia
+#
+# Essentia is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Affero General Public License as published by the Free
+# Software Foundation (FSF), either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the Affero GNU General Public License
+# version 3 along with this program. If not, see http://www.gnu.org/licenses/
+
+from argparse import ArgumentParser
+from multiprocessing import Pool
+from multiprocessing import cpu_count
+from subprocess import Popen, PIPE
+from essentia import EssentiaError
+from functools import partial
+import os
+import sys
+
+
+def __subprocess(cmd, verbose=True):
```

Comment:
I would name this with a single _, normally  2 means that it's internal to python.

the subprocess module has `run` since v 3.5, perhaps we could use this to reduce some of this code? https://docs.python.org/3.7/library/subprocess.html#using-the-subprocess-module